Cyborg upgrade board tweaks

coiax opened the pull request:
:cl: coiax
add: Cyborg renaming boards cannot be used if no name has been entered.
del: Cyborg rename and emergency reboot modules are destroyed upon use,
and not stored inside the cyborg to be ejected if modules are reset.
/:cl:

Reasons: Because being renamed "default name" isn't fun, and it means
you have to hang around to be renamed AGAIN. And for lore/balance/sanity
reasons, you don't get to get the reboot board back after you used it;
try dying less.

coiax committed:
Cyborg upgrade board tweaks

:cl: coiax
add: Cyborg renaming boards cannot be used if no name has been entered.
del: Cyborg rename and emergency reboot modules are destroyed upon use,
and not stored inside the cyborg to be ejected if modules are reset.
/:cl:

Reasons: Because being renamed "default name" isn't fun, and it means
you have to hang around to be renamed AGAIN. And for lore/balance/sanity
reasons, you don't get to get the reboot board back after you used it;
try dying less.

ShadowDeath6 commented:
When is name picking upon activation going to be a thing

ExcessiveUseOfCobblestone commented:
I never understood why midround borgs don't default to the name the person chooses in create character.

jughu commented:
Just make the renameboard with no set name just pick the players preferenced cyborg nsme

SgtxRootbeer commented:
Actually, fuck rename boards, let roboticists use their pen to rename borgs, ala medibot.

coiax committed:
Naming changes, and transformer tweak

- A new unnamed cyborg will copy the user's prefered cyborg name if
possible, defaulting to the old format if there's no preference.
- A rename board with no setting will apply the cyborg's default name,
if one exists, see above.
- Cyborg transformers (from the traitor AI) will no longer destroy all
items.

coiax committed:
Made behaviour more consistent

optimumtact commented:
Please make this work

coiax committed:
Fix spelling

coiax commented:
@optimumtact I'd like to get a test merge of this, it touches stuff that I can't test with just a local client and guest.

RemieRichards commented:
@SgtxRootbeer you can, you just have to name the endoskeleton before you put a brain in it.

optimumtact commented:
@coiax then you need to chase the admins, devs dont' have any power to test merge, except jordie
